src/lib/worker.js

```diff
@@ -325,6 +333,39 @@ var getPodStableNetworkAddressAndPort = function (pod) {
   );
 };
 
+var hasSuffix = function (member) {
+  // console.log("🚀 ~ file: worker.js:341 ~ hasSuffix ~ member:", member)
```

Comment:
I missed this last time

Reply:
would revisit them as of now not using it